ingestion-beam/src/test/java/com/mozilla/telemetry/DecoderTest.java

```diff
@@ -1,323 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was supposed to be deleted in a previous PR where I broke this into separate classes. I somehow screwed that up and this file got included again before merging. Cleaning that up here.